hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/ConsensusServiceSimpleFeesSuite.java

```diff
@@ -0,0 +1,305 @@
+@Tag(MATS)
+@HapiTestLifecycle
+public class ConsensusServiceSimpleFeesSuite {
+
+    @FunctionalInterface
+    public interface OpsProvider {
+        List<SpecOperation> provide();
+    }
+
+    private static CustomSpecAssert validateChargedSimpleFees(
+            String name, String txn, double expectedUsd, double allowedPercentDiff) {
+        return assertionsHold((spec, assertLog) -> {
+            final var actualUsdCharged = getChargedUsed(spec, txn);
+            assertEquals(
+                    expectedUsd,
+                    actualUsdCharged,
+                    (allowedPercentDiff / 100.0) * expectedUsd,
+                    String.format(
+                            "%s: %s fee (%s) more than %.2f percent different than expected!",
+                            name, sdec(actualUsdCharged, 4), txn, allowedPercentDiff));
+        });
+    }
+
+    private static double getChargedUsed(@NonNull final HapiSpec spec, @NonNull final String txn) {
+        requireNonNull(spec);
+        requireNonNull(txn);
+        var subOp = getTxnRecord(txn).logged();
+        allRunFor(spec, subOp);
+        final var rcd = subOp.getResponseRecord();
+        return (1.0 * rcd.getTransactionFee())
+                / ONE_HBAR
+                / rcd.getReceipt().getExchangeRate().getCurrentRate().getHbarEquiv()
+                * rcd.getReceipt().getExchangeRate().getCurrentRate().getCentEquiv()
+                / 100;
+    }
+
+    private Stream<DynamicTest> compareSimpleToOld(
+            OpsProvider provider, String txName, double fee, double simpleDiff, double oldDiff) {
+        List<SpecOperation> opsList = new ArrayList<>();
+
+        opsList.add(overriding("fees.simpleFeesEnabled", "true"));
+        opsList.addAll(provider.provide());
+        opsList.add(validateChargedSimpleFees("Simple Fees", txName, fee, simpleDiff));
+
+        opsList.add(overriding("fees.simpleFeesEnabled", "false"));
+        opsList.addAll(provider.provide());
+        opsList.add(validateChargedSimpleFees("Old Fees", txName, fee, oldDiff));
+
+        return hapiTest(opsList.toArray(new SpecOperation[opsList.size()]));
+    }
+
+    @Nested
+    class TopicFeesComparison {
+        private static final String PAYER = "payer";
+        private static final String ADMIN = "admin";
+
+        @LeakyHapiTest(overrides = {"fees.simpleFeesEnabled"})
+        @DisplayName("compare create topic")
+        final Stream<DynamicTest> createTopicPlainComparison() {
+            return compareSimpleToOld(
+                    () -> Arrays.asList(
+                            cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                            createTopic("testTopic")
+                                    .blankMemo()
+                                    .payingWith(PAYER)
+                                    .fee(ONE_HBAR)
+                                    .via("create-topic-txn")),
+                    "create-topic-txn",
+                    0.01009,
```

Comment:
Why does this have 0.00009 extra for simple fees? It just has one signature for payer and no extras right?

Reply:
Fixed. The base fee was slightly wrong.